### PR TITLE
feat: refactor api for passkey module

### PR DESCRIPTION
**Pull request by bhavanakarwade**
## What ## 
- Refactor API's for passkey model.
- Added success message and refactor typescript issues.
- Handled conditions.
- Resolved UI issues

## Why ## 
- To improve code quality, performance, and bug fixing. It makes the codebase more organized and efficient.
- To enhances user experience

### CODE REVIEW

File: src/components/Authentication/SignInUser.tsx [hunk 1]
```diff
@@ -1,89 +1,102 @@
...
+					fidoFlag === 'true' &&
+					loginMethod === 'passkey'
+				) {
 					setIsPasskeySuccess(true);
 				} else if (resetPasswordSuccess === window?.location?.search) {
 					setSuccess('Congratulations!! 🎉 Your new password set successfully');
 				}
 			} catch (err) {
-				console.log("ERROR-LOGIN::", err)
+				console.log('ERROR-LOGIN::', err);
```
Comment (tipusinghaw) [inline src/components/Authentication/SignInUser.tsx:82]: Please remove the log

File: src/components/Authentication/SignUpUser.tsx [hunk 2]
```diff
@@ -205,7 +205,7 @@ const SignUpUser = () => {
 
 													<Field id='Signupemail'
 														name='email'
-														type="email"
+														// type="email"
```
Comment (tipusinghaw) [inline src/components/Authentication/SignUpUser.tsx:208]: please remove the commented code if not required

File: src/components/Authentication/SignInUser.tsx (hunk 1 above)
Comment (bhavanakarwade) [reply src/components/Authentication/SignInUser.tsx]: resolved

File: src/components/Authentication/SignUpUser.tsx (hunk 2 above)
Comment (bhavanakarwade) [reply src/components/Authentication/SignUpUser.tsx]: resolved

**Pull request merged**